### ISSUE TITLE
Insert script only on windows opened through extension

### ISSUE THREAD

**Issue by tarun14110** [pwablito/2fa_automation#20]
Now the extension is inserting scripts on other windows too. I fixed it a couple of days ago . Probably, the changes during UI abstraction break it.

**Comment by pwablito** [comment]
Could we store a global array of window ids belonging to the automator and check that the id of a window is in that array before injecting scripts?

**Comment by tarun14110** [comment]
Yes! That works, if the insertion and window creation is happening in the background.js. If they are happening in different js, we might have to send the window tab ids whenever we create them. Note: We want to refrain from using browser storage because it could lead to additional permission requirements.
